### PR TITLE
[dev] clean code of s2anet

### PR DESCRIPTION
**Pull request by liuhuiCNN**
(1) clean code of s2anet

### CODE REVIEW

File: ppdet/modeling/heads/s2anet_head.py [hunk 1]
```diff
@@ -23,20 +23,16 @@
 import numpy as np
 
 
-class S2ANetAnchorGenerator(object):
+class S2ANetAnchorGenerator_pd(nn.Layer):
```
Comment (heavengate) [inline ppdet/modeling/heads/s2anet_head.py]: 这个`_pd`是指啥

File: ppdet/modeling/heads/s2anet_head.py [hunk 2]
```diff
@@ -23,20 +23,16 @@
...
     """
-    S2ANetAnchorGenerator by np
+    AnchorGenerator by paddle
```
Comment (heavengate) [inline ppdet/modeling/heads/s2anet_head.py:28]: 注释修改一下

Comment (liuhuiCNN) [reply ppdet/modeling/heads/s2anet_head.py:28]: done

File: ppdet/modeling/heads/s2anet_head.py (hunk 1 above)
Comment (liuhuiCNN) [reply ppdet/modeling/heads/s2anet_head.py]: done

**Review by heavengate — APPROVED** [review]

**Pull request merged**